s3am/Makefile

```diff
@@ -0,0 +1,25 @@
+# Definitions
+build_tool = runtime-container.DONE
+name = quay.io/ucsc_cgl/s3am
+tag = 1.0a1
```

Comment:
@fnothaft I considered containerizing s3am a while ago, but I didn't have a great way to deal with credentials.  You'll need the wrapper script to pull boto from the localhost into ~/.boto inside the container.  

 Also, we include the commit hash so a very specific version of the tool can be referenced, as changes may occur between builds of a container that do not affect the tool version.